oldurls.json

```
@@ -138,6 +138,9 @@
   "runtime/reference/deno_namespace_apis/": "/api/deno/",
   "runtime/reference/web_platform_apis/": "/api/web/",
   "runtime/reference/node_apis/": "/api/node/",
-  "/deploy/tutorials/tutorial-http-server/": "/examples/http_server/",
-  "/deploy/manual/*": "/deploy/classic/*"
+  "/deploy/tutorials/vite/": "/examples/react_tutorial/",
+  "/deploy/manual/*": "/deploy/classic/*",
+  "/deploy/early-access/*": "/deploy/*",
+  "/deploy/tutorials/": "/examples/",
+  "/deploy/tutorials/*": "/examples/"
```

Not sure of the intended logic here, but just to double check that this shouldn't pass the wildcard through.

Should this one be this?

```suggestion
  "/deploy/tutorials/*": "/examples/*"
```